package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "prop-types": "^15.7.2",
```

Comment:
is this necessary?

Reply:
this is for props validation and it is requested my the linting tool